recipe/bld.bat

```diff
@@ -5,23 +5,34 @@ if %ARCH%==32 (
 )
 
 set EPICS_BASE=%PREFIX%\epics
+echo INSTALL_LOCATION = %EPICS_BASE:\=/%> configure\CONFIG_SITE.local
```

Comment:
I have no idea what this line does. What is  `%EPICS_BASE:\=/%`?

This is clearly a parallel to
```py
RELATIVE_INSTALL_LOCATION = Path(os.path.relpath(EPICS_BASE))
Path("configure/CONFIG_SITE.local").write_text(
    f"INSTALL_LOCATION = $(TOP)/{RELATIVE_INSTALL_LOCATION.as_posix()}\n"
)
```
and I am _guessing_ it's meant to take EPICS_BASE and swap the slashes?

Actually wtf is going on with that prebuild script? Why does it set the install location to `$(TOP)/epics`? Is that actually the same as `${PREFIX}/epics`?

Reply:
> I have no idea what this line does. What is %EPICS_BASE:\=/%?

Windows batch variable substitution: `%var:from=to%`